docker/linksets/asgs2geofab/start.sh

```diff
@@ -3,4 +3,4 @@ set -e
 
 source ../../../common/common.sh
 
-docker-compose up -d --build --force-recreate
+docker-compose up -f docker-compose.base.yml -d --build --force-recreate
```

Comment:
Unsure what running base, on its own would achieve. Should this script be removed?